pkg/build/apiserver/registry/buildlog/rest.go

```diff
@@ -48,11 +50,11 @@ const defaultTimeout time.Duration = 30 * time.Second
 // NewREST creates a new REST for BuildLog
 // Takes build registry and pod client to get necessary attributes to assemble
 // URL to which the request shall be redirected in order to get build logs.
-func NewREST(buildClient buildtypedclient.BuildsGetter, podClient kubetypedclient.PodsGetter) *REST {
+func NewREST(buildClient buildtypedclient.BuildsGetter, podClient kubetypedclient.PodsGetter, podLister corev1listers.PodLister) *REST {
 	r := &REST{
-		BuildClient: buildClient,
-		PodClient:   podClient,
-		Timeout:     defaultTimeout,
+		PodClient: podClient,
+		PodLister: podLister,
+		Timeout:   defaultTimeout,
```

Comment:
Missed setting the BuildClient here - this is probably why all build tests are bombing out.

```suggestion
		BuildClient: buildClient,
		PodClient:   podClient,
		PodLister:   podLister,
		Timeout:     defaultTimeout,
```